File: src/test/java/kitchenpos/fixture/MenuGroupFixtures.java
package kitchenpos.fixture;

import kitchenpos.domain.MenuGroup;

import java.util.UUID;

public class MenuGroupFixtures {

    public static MenuGroup createMenuGroup(String name) {
        MenuGroup menuGroup = new MenuGroup();
        menuGroup.setId(UUID.randomUUID());
        menuGroup.setName(name);
        return menuGroup;
    }

    public static MenuGroup createMenuGroup() {
        MenuGroup menuGroup = new MenuGroup();
        menuGroup.setId(UUID.randomUUID());
        menuGroup.setName("메뉴그룹");
        return menuGroup;
    }
}

File: src/test/java/kitchenpos/testsupport/MenuGroupFixtures.kt
package kitchenpos.testsupport

import java.util.UUID
import kitchenpos.domain.MenuGroup

object MenuGroupFixtures {
    fun createMenuGroup(
        id: UUID? = UUID.randomUUID(),
        name: String? = "test-menu-group-name"
    ): MenuGroup {
        return MenuGroup().apply {
            this.id = id
            this.name = name

        }
    }
}